### PR TITLE
Applepay

### PR DESCRIPTION
**Pull request by asood123**

### CODE REVIEW

File: frontend/src/components/Tiers.js [hunk 1]
```diff
@@ -107,23 +124,42 @@ export default class Tiers extends Component {
...
-                description={stripeDescription}>
-                  <AsyncButton
-                    color='green'
-                    inProgress={inProgress} >
-                    {button}
-                  </AsyncButton>
-              </StripeCheckout>
-            )}
 
+            {hasStripe && !hasPaypal && (
```
Comment (xdamman) [inline frontend/src/components/Tiers.js]: Do we really need `!hasPaypal`  here?

File: frontend/src/components/Tiers.js [hunk 2]
```diff
@@ -152,6 +188,37 @@ export default class Tiers extends Component {
       </div>
     );
   }
+
+  callApplePay(tier, payload) {
+    const { collective, onToken } = this.props;
+
+    // setup the payment request
+    const paymentRequest = {
+      countryCode: 'US', // TODO: find out if we need to change this for other countries
```
Comment (xdamman) [inline frontend/src/components/Tiers.js]: Should this be the country of the platform, or of the Stripe account of the host?
(I'd imagine the latter)

File: frontend/src/components/Tiers.js [hunk 3]
```diff
@@ -152,6 +188,37 @@ export default class Tiers extends Component {
...
+      currencyCode: collective.currency,
+      total: {
+        label: collective.name,
```
Comment (xdamman) [inline frontend/src/components/Tiers.js]: shouldn't the label be something more explicit (including the `tier.description`)?

**Comment by xdamman** [comment]
Great job on this! Can't wait to push this live!

File: frontend/src/components/Tiers.js (hunk 1 above)
Comment (asood123) [inline frontend/src/components/Tiers.js]: don't think we need it either - didn't want to touch the old logic. It's more likely to cause bugs and we are going to retiring this code base anyway.

File: frontend/src/components/Tiers.js (hunk 2 above)
Comment (asood123) [inline frontend/src/components/Tiers.js]: unclear from docs. I'll try some variations.

File: frontend/src/components/Tiers.js (hunk 3 above)
Comment (asood123) [inline frontend/src/components/Tiers.js]: changed

**Pull request merged**